test/lib.error-handler.spec.js

```diff
@@ -21,8 +21,9 @@ describe('lib/error-handler', () => {
 
     const err = await throwsAsync(errorHandler(null, null, res))
 
+    expect(err instanceof HTTPError).to.be.true()
     expect(err.message).to.eql('boom')
-    expect(err.status).to.eql(500)
+    expect(err.response.status).to.eql(500)
```

Comment:
Breaking change?  I guess errors aren't really part of the spec other than 'you will get an error'.

Reply:
Yes, it is breaking. I've updated the PR desctiption with the info and will include it when merging.

Reply:
Great.  One small thing - the description says `err.status` has changed but it's `err.statusCode`

Reply:
Well this test was looking for `.status` but others are looking for `.statusCode`...I've added both to the breaking change message.